build/_build.csproj

```diff
@@ -2,15 +2,21 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
     <NoWarn>CS0649;CS0169</NoWarn>
     <NukeRootDirectory>..</NukeRootDirectory>
     <NukeScriptDirectory>..</NukeScriptDirectory>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Nuke.Common" Version="0.24.11" />
+    <PackageReference Include="Nuke.Common" Version="5.1.0" />
     <PackageDownload Include="GitVersion.Tool" Version="[5.1.1]" />
```

Comment:
Did this also need to be updated? Still failing
https://build.octopushq.com/buildConfiguration/Octofront_Octopurls_BuildAndDeployNew/2257100